disnake/webhook/async_.py

```diff
@@ -539,8 +540,12 @@ def handle_message_parameters_dict(
         payload["avatar_url"] = str(avatar_url)
     if username:
         payload["username"] = username
+
+    payload["flags"] = 0
```

Comment:
shouldn't put flags into the payload unless necessary, since this method is being used by edit methods as well

Reply:
https://github.com/DisnakeDev/disnake/pull/474/commits/b51822370ae36a8b9eb9592d4dfc82885beec7ff